snap/snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: prometheus-bind-exporter
+version: git
+title: Prometheus bind exporter
+summary: Export BIND (named/dns) v9+ service metrics to Prometheus.
+description: Export BIND (named/dns) v9+ service metrics to Prometheus.
+
+grade: stable
+confinement: classic
```

Comment:
This needs to be `strict`. Classic "confinements" (unconfined, really) is supported for applications who can't work on strictly confined snap mode (snap interfaces missing or other reasons).